contact.md

```diff
@@ -4,14 +4,20 @@ title: Contact
 permalink: /contact/
 ---
 
-E-Mail: [info@ayab-knitting.com](mailto:info@ayab-knitting.com)
+### Contribution, Discussion, Support
+
+Our main place for discussion about the AYAB project is the [All Yarns Are Beautiful Discord server](https://discord.gg/tgEM6Df2). \\
```

Comment:
I just tested this invite code on the Discord API (see https://discord.com/api/invite/tgEM6Df2) and it says the invitation expires in one week (`"expires_at": "2024-09-03T19:22:56+00:00" `).

Here's a code that does not expire: https://discord.gg/HdZfuBQWyh , or if you want to generate your own, make sure to change the options when generating the link.

Reply:
🤦 thanks for the hint...will change